.github/workflows/test.yml

```diff
@@ -153,17 +174,28 @@ jobs:
           username: ${{ github.actor }}
           password: ${{ secrets.GITHUB_TOKEN }}
 
+      - name: Set up path
+        run: |
+          mkdir -p "$HOME/.local/bin"
+          echo "$HOME/.local/bin" >> "$GITHUB_PATH"
+
       - name: Install Hurl
         env:
           VERSION: '5.0.1'
+          ARCH: ${{ runner.arch == 'X64' && 'x86_64' || 'aarch64' }}
         run: |
-          curl --location --remote-name https://github.com/Orange-OpenSource/hurl/releases/download/$VERSION/hurl_${VERSION}_amd64.deb
-          sudo apt update && sudo apt install ./hurl_${VERSION}_amd64.deb
+          curl --location --remote-name https://github.com/Orange-OpenSource/hurl/releases/download/$VERSION/hurl_${VERSION}_arm64.deb
+          
```

Comment:
Now there's a `.deb` *and* a `.tar.gz` downloaded? Also, the DEB should work like before with just a mapping of `x86_64 => amd64` and `aarch64 => arm64` for the filename/URL.

Reply:
Just tried it on both archs, a simple way to get the needed string would be to use `dpkg --print-architecture`.

Reply:
Ah good catch, the .deb should be removed: there is no asset for the ARM64 architecture (https://github.com/Orange-OpenSource/hurl/releases/tag/5.0.1).

But I just saw newer releases do have an ARM64 .deb, so that is a lot easier (https://github.com/Orange-OpenSource/hurl/releases/tag/6.1.1).

We can probably upgrade Hurl